src/run.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from 'fs-extra';
+import * as globby from 'globby';
+import * as mkdirp from 'mkdirp';
+import { join, basename } from 'path';
+
+import WidgetDataInterface from './WidgetDataInterface';
+import createThemeFile from './createThemeFile';
+import convertSelectorsToCSS from './convertSelectorsToCSS';
+import {
+	packageQuestions,
+	getFileQuestions,
+	askForDesiredFiles,
+	askForPackageName
+} from './questions';
+
+async function run() {
+	const themesDirectory = 'src/themes';
+	const packageName = await askForPackageName(packageQuestions);
+
+	const selectedpackagePath = join('node_modules', packageName, 'theme');
+
+	if (!fs.existsSync(selectedpackagePath)) {
+		throw new Error(`This package path does not exist: ${selectedpackagePath}`);
+	}
+
+	const cssDataFileExtension = '.m.css.js';
+	const CSSModuleExtension = '.m.css';
+	const cssDataFileGlob = join(selectedpackagePath, `**/*${cssDataFileExtension}`);
+	const matchingCSSFiles = globby.sync(cssDataFileGlob);
+
+	const fileQuestions = getFileQuestions(packageName, matchingCSSFiles, cssDataFileExtension);
+	const selectedWidgets = await askForDesiredFiles(fileQuestions);
+
+	const themedWidgets = selectedWidgets.map((selectedWidget: string): WidgetDataInterface => {
+		const [fileName] = basename(selectedWidget).split(cssDataFileExtension);
```

Comment:
just a nit for code style, can you put spaces inside the array please? ie. `const [ filename ] =`

Reply:
Done!